packages/utilities/etc/utilities.api.md

```diff
@@ -330,7 +330,7 @@ export function getLastFocusable(rootElement: HTMLElement, currentElement: HTMLE
 export function getLastTabbable(rootElement: HTMLElement, currentElement: HTMLElement, includeElementsInFocusZones?: boolean, checkNode?: boolean): HTMLElement | null;
 
 // @public
-export function getNativeElementProps<TAttributes extends React.HTMLAttributes<any>>(tagName: keyof React.ReactHTML, props: {}, excludedPropNames?: string[]): TAttributes;
+export function getNativeElementProps<TAttributes extends React.HTMLAttributes<any>>(tagName: string, props: {}, excludedPropNames?: string[]): TAttributes;

```

Comment:
Why was this change?

Reply:
In usage the restriction of only allowing ReactHTML keys fought with the incoming typings, so I'd find myself casting tagname to `any` or `as unknown as React.ReactHTML` just to coerce things. 

The function can support any string. You can pass "foobar" to it which in the web components world could be a valid element, and it will still give you basic baseline filtered attributes.